portafly/src/i18n/locales/en/accounts.json

```diff
@@ -0,0 +1,12 @@
+{
+  "page_title": "Accounts | Portafly",
```

Comment:
We need a description for every field
```
page_title_description
body_title_description
```
etc

Reply:
I wouldn't worry too much about it right now, it's just a dummy example of Accounts page.